components/authz-service/server/v2/authz_test.go

```diff
@@ -78,10 +79,10 @@ func TestV2p1ProjectsAuthorized(t *testing.T) {
 				[]string{},    // no projects
 				[]string(nil), // well, almost verbatim
 			},
-			"when the request includes SOME projects and the engine response is ALL projects, returns the requested projects": {
+			"when the request includes SOME projects and the engine response is ALL projects, returns external ALL projects": {
 				[]string{"p1", "p2", "p3"},
 				[]string{constants.AllProjectsID, "p3"},
-				[]string{"p1", "p2", "p3"},
+				[]string{constants.AllProjectsExternalID},
```

Comment:
I was about to say this test result is wrong--it should not have changed.
But then I realized why. When I wrote this set of tests originally, I gave them the wrong set of inputs. They are testing _only_ what authz does with the OPA output. The requested projects should not be given as an input to these tests at all.
So if you don't mind cleaning these up while you're here, that would be useful.
(E.g. This test name should become "when then engine response is ALL projects, returns external ALL projects"